frontend/cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file required?

Reply:
Not jet, but thats where the `baseUrl` is supposed to be i guess ;)